components/Button.stories.ts

```diff
@@ -31,16 +29,56 @@ export default {
       },
       control: { type: 'radio' },
     },
+    isDisabled: {
+      name: 'Disabled',
+      type: { name: 'boolean', required: false },
+      defaultValue: false,
+      description: 'Disable interaction if set to true',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+      control: {
+        type: 'boolean',
+      },
+    },
   },
 }
 
-const Template = (_args: any, { argTypes }: any) => ({
+export const Default = (_args: any, { argTypes }: any) => ({
+  props: Object.keys(argTypes),
+  template: `
+    <div class="w-[300px]">
+      <Button v-bind="$props">
+        {{title}}
```

Comment:
Please add space between JS value
```suggestion
        {{ title }}
```